resources/sass/materialize.scss

```diff
@@ -21,6 +21,39 @@ $link-color: color("coli", "orange");
 //(will include the previous ones also, but it won't overwrite those)
 @import "node_modules/materialize-css/sass/materialize.scss";
 
+
+@mixin column-count($ct) {
+  -webkit-column-count: $ct;
+  -moz-column-count: $ct;
+  column-count: $ct;
+}
+
+.cards-container {
+  column-break-inside: avoid;
+  .card {
+    display: inline-block;
+    overflow: visible;
+  }
+}
+
+@media #{$small-and-down} {
+  .cards-container {
+    @include column-count(1);
+  }
+}
+@media #{$medium-and-up} {
+  .cards-container {
+    @include column-count(2);
+  }
+}
+@media #{$large-and-up} {
+  .cards-container {
+    @include column-count(3);
+  }
+}
```

Comment:
These meant to create nice and fluid divs filled with cards with variable height in more columns.
It was buggy here (I think because I use overflow property for something else), however, we can use this elsewhere in the future.